db/fixtures/ae_datastore/ManageIQ/Service/Retirement/StateMachines/ServiceRetirement.class/__methods__/update_service_retirement_status.rb

```diff
@@ -22,12 +22,20 @@
 $evm.log("info", "Server:<#{server.name}> Ae_Result:<#{$evm.root['ae_result']}> State:<#{state}> Step:<#{step}>")
 $evm.log("info", "Status_State:<#{status_state}> Status:<#{status}>")
 
+# Update Status Message
+updated_message  = "Server [#{server.name}] "
+updated_message += "Service [#{service.name}] " if service
+updated_message += "Step [#{step}] "
+updated_message += "Status [#{status}] "
+updated_message += "Current Retry Number [#{$evm.root['ae_state_retries']}]" if $evm.root['ae_result'] == 'retry'
+
 # Update Status for on_error for all states other than the first state which is start retirement
 # in the retirement state machine.
 if $evm.root['ae_result'] == 'error'
   if step.downcase == 'start retirement'
     $evm.log("info", "Cannot continue because Service is already retired or is being retired.")
```

Comment:
@billfitzgerald0120 Can we add a "warn" notification for the case where the VM is already retired or is being retired?